imap-proto/src/builders/command.rs

```diff
@@ -1,5 +1,7 @@
-use crate::quoted_string;
-use crate::types::{AttrMacro, Attribute, State};
+use crate::{
```

Comment:
You did a bunch of these earlier, this should be probably be in the same commit.